corehq/apps/app_manager/dbaccessors.py

```diff
@@ -22,14 +22,14 @@ def domain_has_apps(domain):
     return len(results) > 0
 
 
-def get_latest_released_app_doc(domain, app_id, min_version=None):
+def get_latest_released_app_doc(domain, app_id):
```

Comment:
No, that makes a lot of sense, I was focusing on the wrong part of it. You're saying that `min_version` is always, as it's used here, going to be set to the version of the build that would have been selected anyway. If that's the case (which I agree it appears to be) then I'm totally with you. Thanks for walking me through it.